deep-learning/_posts/2017-02-27-unet.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: review
+title: U-Net Convolutional Networks for Biomedical Image Segmentation
+tags: deep-learning CNN segmentation medical essentials
+cite:
+  - Unet 
+  - https://arxiv.org/pdf/1505.04597.pdf
+---
+
+<style>
+  main.page-content p {
+    text-align: justify; /* helps the reading flow */
+  }
+</style>
+
+
+O. Ronneberger, P. Fischer, T. Brox **U-Net: Convolutional Networks for Biomedical Image Segmentation**, in proc of MICCAI 2015, p.234-241.
+
+[https://arxiv.org/pdf/1505.04597.pdf](https://arxiv.org/pdf/1505.04597.pdf)
```

Comment:
Ne need to add it there, the `cite:` statement automaticaly insert the link to the paper in the page.
The `cite` changes are in the BatchNorm PR (#5).